index.html

```diff
@@ -289,6 +289,20 @@ <h2>About me</h2>
           </ul>
         </div>
       </div>
+      <div class="contact-form">
```

Comment:
- [ ]  ( optional ) I think it would be better to use "< section >" tag instead of < div > for your code to be more semantic